pubspec.yaml

```diff
@@ -4,13 +4,13 @@ version: 0.0.1
 homepage: https://github.com/icapps/flutter-custom-image-crop
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
-  flutter: ">=1.17.0"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
```

Comment:
I would target this on 2.0.2 because of a breaking issue in flutter 2.0.0 that was patched in 2.0.2
https://github.com/flutter/engine/pull/24928